internal/action/git.go

```diff
@@ -0,0 +1,35 @@
+package action
+
+import (
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/gopasspw/gopass/internal/out"
+	"github.com/gopasspw/gopass/pkg/ctxutil"
+	"github.com/urfave/cli/v2"
+)
+
+// Git passes the git command to the underlying backend.
+func (s *Action) Git(c *cli.Context) error {
+	ctx := ctxutil.WithGlobalFlags(c)
+	store := c.String("store")
+
+	sub, err := s.Store.GetSubStore(store)
+	if err != nil {
+		return ExitError(ExitGit, err, "failed to get sub store %s: %s", store, err)
+	}
+	if sub == nil {
+		panic("boom")
```

Comment:
This seems not super descriptive. Could it happen on a non-initialized setup, maybe? Or is it really supposed to never happen?

Reply:
Haha, sorry. Forgot this